### ISSUE TITLE
NVDA/Narrator is not announcing Search results whether result is found or not.

### ISSUE THREAD

**Issue by msftedad** [ag-grid/angular-data-grid#7]
**Environment (OS, Application, Versions):** 
OS: Windows 11 Enterprise 23H2 (22631.3737)
Edge: Version 126.0.2592.87 (Official build) (64-bit)

**Repro Steps:**

1.Now turn on NVDA/Narrator
2.Open this url "[app.component.ts - nodebox - CodeSandbox](https://codesandbox.io/p/sandbox/ag-grid-packages-6769q4?file=%2fapp%2fapp.component.ts)".
3.Tab till preview tab which is present at right down corner.
4.Tab till quick filter input box and press enter.
5.Observe that NVDA/Narrator is not announcing Search results whether result is found or not.

**Actual Results:** 

Narrator/NVDA is not announcing Search results whether result is found or not.

**Expected Results:**

Narrator/NVDA should announce Search results whether result is found or not.

https://github.com/user-attachments/assets/ebee9064-4d03-4dcb-b965-6591a4765258

https://github.com/user-attachments/assets/19e2c9e3-8c32-4131-848b-f2826b1e7a96
